src/cinder/audio/linux/FileAudioLoader.cpp

```diff
@@ -128,6 +128,9 @@ struct IStreamSndFile {
 struct IStreamMpg123 {
 	static ssize_t read( void* userData, void* ptr, size_t count ) {
 		ci::IStreamCinder* stream = static_cast<ci::IStreamCinder*>( userData );
+		if (stream->isEof()) {
```

Comment:
Did you encounter an issue with this? Not sure if `read()` returning zero bytes during a failure scenario is an ideal solution, but would like to understand how this came about.